### PR TITLE
add code smell and technical debt section

### PR DESCRIPTION
**Pull request by SteveBurgos95**

### CODE REVIEW

File: 4-DevOps/Reportings/Vanquish/Reports.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- The code should become cleaner.
+- New functionality should not be created
+- All existing tests must pass after refactoring
+
+
+# <strong>Code Smells</strong>
+
+<strong>According to Martin Fowler</strong>, <i>"a code smell is a surface indication that usually corresponds to a deeper problem in the system."</i>
+
+# Types of Code Smells
```
Comment (kaurrevature) [inline 4-DevOps/Reportings/Vanquish/Reports.md:83]: You might want to explain one liners about the types of code smells if possible

**Review by kaurrevature — CHANGES_REQUESTED** [review]
I also hope some demo would be there to show some code smells

File: 4-DevOps/Reportings/Vanquish/Reports.md (hunk 1 above)
Comment (SteveBurgos95) [reply 4-DevOps/Reportings/Vanquish/Reports.md:83]: Added explanation one liners about the types of code smells

**Review by kaurrevature — CHANGES_REQUESTED** [review]
good jon @SteveBurgos95 
And Kudos to team vanquish!

**Review by kaurrevature — COMMENTED** [review]
Good work

**Pull request merged**